src/content/docs/using-new-relic/cross-product-functions/install-configure/networks.mdx

```diff
@@ -238,6 +238,8 @@ To enhance network performance and [data security](/docs/security/new-relic-secu
   </tbody>
 </table>
 
+### Port 443 recommended [#port-443]
```

Comment:
Add header link

Reply:
Not sure what you mean by this. It already has #port-443, I tested it, and it works. Thanks for clarifying!